ext/webhook.go

```diff
@@ -11,6 +11,8 @@ type WebhookOpts struct {
 	Listen string
 	// Port is the port listen on (eg 443, 8443, etc).
 	Port int
+	// UnixListen is the UNIX socket to listen on (mutually exclusive with Listen/Port options)
```

Comment:
I don't have in mind any other «network type» (transport layer) for HTTP that could be useful here, apart maybe "sctp".

Alternatively we could parse the `Listen` field for the `/` character, as it is invalid in IPv4 and IPv6 addresses notation as well as in DNS hostnames. That would however force to write `./tgbot.sock` instead of just `tgbot.sock` if referring to a socket in the current working directory.